File: StampMachine.cpp
// Stamp Machine
#include <iostream>
using namespace std;

int main()
{
    int dollars;
    int First_Class_Stamps = 47; // 47 cents per first-class stamp
    int Penny_Stamps = 1; //  1 cent per penny stamp
    
    cout << "Welcome to the Stamp Vending Machine!" <<endl<< "Please Enter dollar amount:";
    cin >> dollars;
    First_Class_Stamps = (dollars*100)/47;
    Penny_Stamps = (dollars*100)%47;
    
    cout<< "You have "<< First_Class_Stamps << " First Class Stamps and " << Penny_Stamps << " Penny Stamps" <<endl<< "Thank you!";
    

    return 0;
}

File: StampMachine.py
def stamp_machine(dollars):
    FIRST_CLASS_COST = 47  # 47 cents per first-class stamp
    PENNY_COST = 1  # 1 cent per penny stamp
    
    total_cents = dollars * 100
    first_class_stamps = total_cents // FIRST_CLASS_COST
    remaining_cents = total_cents % FIRST_CLASS_COST
    penny_stamps = remaining_cents // PENNY_COST
    
    print(f"You have {first_class_stamps} First Class Stamps and {penny_stamps} Penny Stamps")
    print("Thank you!")

if __name__ == "__main__":
    dollars = float(input("Welcome to the Stamp Vending Machine!\nPlease Enter dollar amount: "))
    stamp_machine(dollars)
